.github/workflows/check-docker-version.yaml

```diff
@@ -0,0 +1,33 @@
+# This workflow works with bump-version.yaml to ensure that the version in the
+# Dockerfile is consistent with what's in the version file. If not, it aborts the
+# workflow so a human can fix it. This is meant to run on every commit to a PR.
+# The dockervar should appear near the top of the file in a statement like:
+# ARG FOO_VERSION=1.2.3
+# The actual version should just be a semver version with no -buildnumber and no leading "v" or other characters.
+
+name: Check Docker Version
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+"on":
```

Comment:
Craig was removing the quotes from these `on` lines.

Reply:
It works either way, but the quotes are leftover from when we were patching these files and are weird now

Reply:
I agree with Craig; it should be fine to remove the quotes, and it's a bit more readable. I can supply lots of detail about YAML parser bugs that made the quotes necessary when we were patching, but you probably don't want to hear it. tl;dr: YAML kinda sucks.

Reply:
Consistency seems beneficial to me.